code/game/machinery/doors/airlock.dm

```diff
@@ -30,12 +30,14 @@
 	var/hasShocked = 0 //Prevents multiple shocks from happening
 	var/secured_wires = 0
 	var/datum/wires/airlock/wires = null
+	waterproof = 0
 
 	var/open_sound_powered = 'sound/machines/airlock.ogg'
 	var/open_sound_unpowered = 'sound/machines/airlock_creaking.ogg'
 
 	var/_wifi_id
 	var/datum/wifi/receiver/button/door/wifi_receiver
+	var/_airlock_function

```

Comment:
Not that I'm aware of. I tend to use them as "initial configuration" vars as they are pushed to the top of the variable list in Dream Maker.